docs/tools/swift.pod

```diff
@@ -1,30 +1,114 @@
 =pod
 
+=encoding utf8
+
 =head1 NAME
 
-swift - the amazingly new programming language
+swift - General purpose programming language
 
 =head1 SYNOPSIS
 
-B<swift> [-help] [I<input-filename> [I<program-arguments>]]
+To invoke the Swift REPL (Read-Eval-Print-Loop):
+
+=over
+
+B<swift>
+
+=back
+
+To execute a Swift program:
+
+=over
+
+B<swift> program.swift -- <arguments>
+
+=back
+
+To work with the Swift Package Manager:
+
+=over
+
+B<swift> build|package|run|test [options]
+
+=back
+
+To invoke the Swift Compiler:
```

Comment:
Although "Swift Package Manager" is capitalized throughout, here it's just "Swift compiler" without a capital C (as evidenced by the style used in `swift -help` and `swift package -help`).